nn/conformer.py

```diff
@@ -187,7 +187,7 @@ def __init__(self, encoder_layer: nn.Module, num_blocks: int, conv_kernel_size:
 
     self.linear = nn.Linear(n_out=enc_key_dim, l2=l2, with_bias=False)
 
-    self.conformer_blocks = nn.ModuleList([
+    self.conformer_blocks = nn.Sequential([
```

Comment:
For `Sequential`, you don't need to pass it as a `list`, you can directly pass it as the arguments.
```suggestion
    self.conformer_blocks = nn.Sequential(
```

Reply:
Although, this is slightly different, it will be an iterable here, but anyway, this should also work. If it does not, we should fix `Sequential` such that it works.

Reply:
You still have the `[`.